Libraries/Components/Touchable/Touchable.js

```diff
@@ -325,9 +325,17 @@ const TouchableMixin = {
       evt.dispatchConfig = {};
       if (myTag === evt.tag) {
         if (evt.eventType === 'focus') {
-          cmp.touchableHandleActivePressIn && cmp.touchableHandleActivePressIn(evt);
+          if (cmp.touchableHandleActivePressIn) {
```

Comment:
>touchableHandleActivePressIn [](start = 18, length = 28)

Would this preclude us from supporting keyboard navigation scenarios. E.g., get a focus event when tabbing to and pressIn event when you hit Enter or space?

Reply:
To try and minimize controversy this PR deprecates the behavior of onPressIn/Out being called for focus events on TVs, with the goal of eventually removing it so that on platforms where onFocus _and_ onPressIn make sense it would be supported. As far as Enter/space specifically, I believe that'd be a platform-specific behavior, e.g. on the web hitting space inside of an input does not by default do anything other than add a space character to the input value.

Does that make sense?

Reply:
So eventually, TouchableHighlight and TouchableOpacity would not work on tvOS?

Reply:
@dlowder-salesforce oops that's not the intent, can you elaborate?

Reply:
Currently in tvOS, a focus event fires onPressIn, a blur event fires onPressOut.  This allows TouchableHighlight and TouchableOpacity to work very naturally without changes on tvOS, and therefore an iOS and tvOS app can share a lot of React Native UI code.  It seemed to me that the changes in this PR were heading in a direction where this would eventually not be possible -- did I misunderstand what you are trying to do?

Reply:
@dlowder-salesforce how do you differentiate between focus and holding a button on the remote? I would think the onFocus event would trigger a different highlight than holding a button down.

Reply:
@rozele on Apple TV, we don't do any special highlighting when the remote button is held down.  In general, TV apps highlight a view only when it comes into focus as a result of arrow key or touchpad swipe navigation.